poc/vdaf_mastic.py

```diff
@@ -180,8 +171,7 @@ def prep_init(cls, verify_key, agg_id, agg_param,
                   nonce, public_share, input_share):
         (level, prefixes, do_range_check) = agg_param
         (vidpf_init_seed, flp_proof_share, flp_blind) = \
```

Comment:
Change this to (vidpf_init_seed, flp_proof_share, flp_seed) since we've gotten rid of the "blinds"?